app/views/docs/functions-deploy.phtml

```diff
@@ -0,0 +1,198 @@
+    You can automatically deploy Appwrite Functions from source control, build your own deployment pipelines using the Appwrite CLI, or upload code files manually.
+    Here's everything you need to know to deploy your first Appwrite Function.
+</p>
+
+<h2><a href="#vcs" id="vcs">VCS (Version Control System)</a></h2>
+<p>
+    The recommended way to manage your Appwrite Function deployments is to use a version control system, like Git.
+    This offers simple versioning and collaboration that will easily fit into the rest of your development workflow.
+</p>
+
+<h3>Create Funcion</h3>
+<p>Before deploying your function with Git, create a new function attached to your Git repo.</p>
+<ol class="margin-top margin-bottom-large text-size-normal">
+    <li>
+        Navigate to <b>Functions</b> from the side bar of the Appwrite Console.
+    </li>
+    <li>
+        Click <b>Create function</b>.
+    </li>
+    <li>
+        When asked to <b>Choose your source</b>, under <b>Connect Git repository</b>, select your provider.
+    </li>
+    <li>
+        Search for the Git repository that hold your function and click <b>connect</b>.
+    </li>
+    <li>
+        Select a production branch. New commits made to the production branch will be <b>automatically deployed and activated</b>.
+    </li>
+    <li>
+        Input the root directory of the function inside the repository. 
+    </li>
+    <li>
+        If you don't want deploy comments to be made on your PRs, select <b>Silent mode</b>.
+    </li>
+    <li>
+        Name your function, select a runtime that matches your function, and enter an entry point path, relative to the root directory from the previous step.
+    </li>
+    <li>
+        If you have build steps, like installing dependencies, input the commands into the <b>Build settings</b> heading's <b>Command</b> field.
+    </li>
+    <li>
+        Finally, configure the execute permissions of the function. For security, only provide execute permissions to the necessary roles.
+    </li>
+</ol>
+
+<h3>Deploy</h3>
+<ol class="margin-top margin-bottom-large text-size-normal">
+    <li>
+        Checkout your prodction branch in Git. 
+    </li>
+    <li>
+        Create a new commit.
+    </li>
+    <li>
+        Push your new commit.
+    </li>
+    <li>
+        A new deployment will be automatically created. Deployments will be automatically activated when new commits are added to the production branch.
+    </li>
+</ol>
+<h3>Debugging Builds</h3>
+<p>After deploying a function, you can find the status of the deployment and build logs in the Appwrite Console.</p>
+<ol class="margin-top margin-bottom-large text-size-normal">
+    <li>In Appwrite Console, navigate to <b>Functions</b>.</li>
+    <li>Click to open a function you wish to inspect.</li>
+    <li>Under the <b>Deployments</b> tab, you'll find the status of the current active deployment and previous inactive deployments.</li>
```

Comment:
Should we also add what `redeploy` function does for both manual and VCS deployments?

Example: 
- For VCS deployments, redeploy uses the same commit hash but updated function settings.
- For manual deployments, redeploy uses previous code but updated function settings.